packages/svelte/src/compiler/compile/index.js

```diff
@@ -116,6 +116,8 @@ function validate_options(options, warnings) {
 /**
  * @param {string} source
  * @param {import('../interfaces.js').CompileOptions} options
+ *
+ * https://svelte.dev/docs/svelte-compiler#svelte-compile
```

Comment:
I'm surprised this is below the `@param` tags. Doesn't the description usually go above the tags? I also wonder if we need a blank line as I think we don't normally have a blank line between things in the JSDoc comments

Reply:
Generally yes, it should be above the tags, but i was following the convention of svelte modules's links

I can move em all to be above the tags tomorrow.

As for the space, we should have space between description and links if possible. Just my thoughts, no design principles on this i believe.

Reply:
yeah, I guess if there's a real description putting a blank line between it and the link could make sense. I probably wouldn't put one between the link and `@param`

Reply:
Updated! Also took the liberty of adding a few excerpts from the documentation for each function's description, so the user doesn't have to leave the IDE to come to the website until unless they wanna do a more complex thing

One exception is for transitions, where I added their parameters and default values too. This helps immensely, as before I never really knew what durations the transitions would run for, or what easing were they using. This change will alleviate that for other users as well